tutor-contrib-harmony-plugin/setup.py

```diff
@@ -26,7 +26,7 @@ def load_about():
 
 
 setup(
-    name="tutor-contrib-harmony-plugin",
+    name="tutor-contrib-harmony",
```

Comment:
This is just to follow the standard on tutor plugins. Eventually, we would want to rename the folders but that's a large PR just to introduce this workflow. It's better to have a different PR for that